roll/testing.py

```diff
@@ -25,6 +79,30 @@ class Client:
     def __init__(self, app):
         self.app = app
 
+    def handle_files(self, kwargs):
```

Comment:
Not related to this PR, but let's have a dedicated doc page (how-to) for "testing".

Reply:
Done!